sdk/keyvault/Azure.Security.KeyVault.Certificates/src/DeleteCertificateOperation.cs

```diff
@@ -24,17 +24,20 @@ internal DeleteCertificateOperation(KeyVaultPipeline pipeline, Response<DeletedC
         {
             _pipeline = pipeline;
             _value = response.Value ?? throw new InvalidOperationException("The response does not contain a value.");
-            _operationInternal = new(_pipeline.Diagnostics, this, response.GetRawResponse(), nameof(DeleteCertificateOperation), new[]
-            {
-                new KeyValuePair<string, string>("secret", _value.Name), // Retained for backward compatibility.
-                new KeyValuePair<string, string>("certificate", _value.Name),
-            });
 
             // The recoveryId is only returned if soft delete is enabled.
             if (_value.RecoveryId is null)
             {
                 // If soft delete is not enabled, deleting is immediate so set success accordingly.
-                _operationInternal.SetState(OperationState.Success(response.GetRawResponse()));
+                _operationInternal = OperationInternal.Succeeded(response.GetRawResponse());
```

Comment:
The variables have been dropped and should be retained.

Reply:
Same in the two other instances below.

Reply:
Sorry, can you please clarify this? Which variables have been dropped?

Reply:
Before, the "secret" and "certificate" variables were added invariably:
<img width="420" alt="image" src="https://user-images.githubusercontent.com/1532486/165359432-f95e9446-0b67-4b4e-8ff6-c2cf63c475b6.png">
But now they are only added if `RecoveryId` is null. One could argue that they are necessary if there's nothing to do, but I do worry about a regression in behavior. @KrzysztofCwalina or @tg-msft what do you think from a breaking change standpoint?

Reply:
When `OperationInternal` is in the final state, no scope will be open, so these scope attributes will never be used. This is different from the previous behavior when calling `UpdateStatusAsync` would make at least one call to the server. However, that call would also override the response assigned by `_operationInternal.SetState`, so I don't think that previous behavior was correct.